gnome/gnome-radio/Portfile

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                gnome-radio
+version             0.1.2
+set branch          [join [lrange [split $version .] 0 1] .]
+
+categories          gnome
+platforms           darwin
+license             GPL-3+
+maintainers         {gnome.org:ole @oleaamot} \
+                    openmaintainer
```

Comment:
Please just use one line for `maintainers`.

Reply:
Wrong, mf2k.  maintainers need two lines, actually.

Reply:
@oleaamot No you don't. Please do what @mf2k asked of you.

Reply:
> Wrong, mf2k. maintainers need two lines, actually.

As both I and @pmetzger have said, that's not true. Just curious, what gives you that idea? If you look at most other `openmaintainer` portfiles, you will see that only one line is used.

Reply:
We prefer that it just be a single line, like this:

`maintainers         {gnome.org:ole @oleaamot} openmaintainer`